### ISSUE TITLE
[antibodies] bugs on serach and display 

### ISSUE THREAD

**Issue by gaetanHerve** [bgo-bioimagerie/platformmanager#709]
cf #702

**Comment by osallou** [comment]
concernant: "4-Pb du texte sur les photos que l'on importe (on est limité en texte contrairement à avant)" 
quel est le pb ? (nom du fichier qui soit etre alphanumerique sans espace ? on ne changera pas)

**Comment by gaetanHerve** [comment]
dommage, j'avais switché de tâche avant de pusher la semaine dernière (après j'avais fait que quelques trucs...)

**Comment by osallou** [comment]
faut pas laisser sans pusher.... mais tu avais du t'attaquer à la recherche non ? (point 1) , je n'y ai pas touché dans cette pr

**Comment by gaetanHerve** [comment]
oui, le 1 notamment. Je regarde l'export en csv en fonction de ce qui est affiché dans le tableau
